{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/urls.py

```diff
@@ -21,7 +21,10 @@
     path('accounts/', include('allauth.urls')),
     path('api/v1/', include('home.api.v1.urls')),
     path('admin/', admin.site.urls),
-    path("users/", include("users.urls", namespace="users")),
+    path('users/', include('users.urls', namespace='users')),
+    path('rest-auth/', include('rest_auth.urls')),
+    path('rest-auth/registration/account-confirm-email/<str:key>/', confirm_email),
```

Comment:
do we need this one? rest auth provide endpoint for email verification.

Reply:
Yes. The default view for account-confirm-email is for API access only. If you click the email link it will error out.

Reply:
What link does it send btw.  if we can configure it to sent the confirmation link send by allauth, then we won't need this.

Reply:
It's sending the allauth link but rest_auth was overriding the HTML view. This overrides the override to put it back to allauth's view.